scripts/wsrep_sst_mysqldump.sh

```diff
@@ -77,7 +77,8 @@ fi
 STOP_WSREP="SET wsrep_on=OFF;"
 
 # NOTE: we don't use --routines here because we're dumping mysql.proc table
-MYSQLDUMP="$MYSQLDUMP $AUTH -S$WSREP_SST_OPT_SOCKET \
+MYSQLDUMP="$MYSQLDUMP --defaults-extra-file=$WSREP_SST_OPT_CONF \
```

Comment:
A cursory examination of wsrep_sst.cc indicates that calls to wsrep_sst_mysqldump will always have a ```--defaults-file``` parameter. I think it is taken from the server variable directly.